pootle/settings/50-project.conf

```diff
@@ -121,3 +121,5 @@ AUTH_USER_MODEL = 'accounts.User'
 COMMENTS_APP = "pootle_comment"
 
 POOTLE_EXPORT_VIEW_LIMIT = 10000
+
+POOTLE_REFRESH_DIRTY_STATS = False
```

Comment:
i think leave the DIRTY - because otherwise it might seem like it disables the refresh_stats command